### ISSUE TITLE
Could not connect to the IPFS API

### ISSUE THREAD

**Issue by octylFractal** [ipfs/ipfs-desktop#2430]
- OS Windows 11
- Version of IPFS Desktop: 0.26.1

**Describe the bug**
A short amount of time (~hours?) after starting IPFS Desktop, the API endpoint will disappear, resulting in "Could not connect to the IPFS API" when the Status view is opened, and local applications being unable to talk with it.

**To Reproduce**
1. Start IPFS Desktop
2. Wait an hour or so
3. Open the Status view.
4. See the error.

**Expected behavior**
The API remains up when IPFS Desktop is running.

**Additional context**
I know that the issue is not an improper URL configuration because shortly after (re)starting the IPFS daemon (not necessairly IPFS Desktop) the API works fine.

[combined.log](https://github.com/ipfs/ipfs-desktop/files/10964833/combined.log)
[error.log](https://github.com/ipfs/ipfs-desktop/files/10964834/error.log)

**Comment by octylFractal** [comment]
The `error.log` is actually empty, unhelpfully. I'll try running kubo directly and report back.

**Comment by octylFractal** [comment]
`kubo` gives me https://github.com/ipfs/kubo/issues/9693 in the output eventually, so it seems it is a `kubo` issue. Should I file a separate issue regarding how the error didn't show up?

**Comment by octylFractal** [comment]
Sadly I don't have capacity for that right now.

**Comment by SgtPooki** [comment]
@octylFractal Are you still running into this issue?

**Comment by octylFractal** [comment]
No, the update to kubo that included the fix for the linked issue fixed it.

**Comment by is-this-echo** [comment]
I am also facing the same issue running ipfs-desktop 0.28.0 with kubo 0.20.0 on Windows 10.
The Status view works fine after I ran ipfs daemon after downloading Windows 10 Kubo binaries.

In my case the error logs are not empty though, sharing them here.
[error.log](https://github.com/ipfs/ipfs-desktop/files/11885446/error.log)
[combined.log](https://github.com/ipfs/ipfs-desktop/files/11885451/combined.log)

**Comment by SgtPooki** [comment]
Thanks for the addition @is-this-echo, I will take a look when I can.
